apis/management.cattle.io/v3/rke_types.go

```diff
@@ -34,6 +34,29 @@ type PrivateRegistry struct {
 	Password string `yaml:"password" json:"password,omitempty"`
 }
 
+type RKESystemImages struct {
+	// etcd image
+	ETCDImage string `yaml:"etcd_image" json:"etcdImage,omitempty" norman:"default=quay.io/coreos/etcd:latest"`
+	// Alpine image
+	AlpineImage string `yaml:"alpine" json:"alpine,omitempty" norman:"default=alpine"`
+	// rke-nginx-proxy image
+	NginxProxyImage string `yaml:"nginx_proxy" json:"nginxProxy,omitempty" norman:"default=rancher/rke-nginx-proxy"`
+	// rke-cert-deployer image
+	CertDownloaderImage string `yaml:"cert_downloader" json:"certDownloader,omitempty" norman:"default=rancher/rke-cert-deployer"`
+	// rke-service-sidekick image
+	ServiceSidekickImage string `yaml:"service_sidekick_image" json:"serviceSideKickImage,omitempty" norman:"default=rancher/rke-service-sidekick"`
```

Comment:
I'd also chose more descriptive naming for this image. ServiceSidekick is a bit generic; user won't have any idea what it's for

Reply:
Also would break compatibility if changed.

Reply:
Again, it is fine, we can release note it.